Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+String cron_string = BRANCH_NAME == "develop" ? "H H(0-8) * * *" : ""
```

Comment:
Should we add a shebang?

```
#!/usr/bin/env groovy
```

Reply:
This is not required, but let's add this